src/basic/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Basic Auth plugin.
+ */
+class BasicAuth {
+  constructor({ loginPath = '/login', logoutPath = '/logout', fetchUser = null } = {}) {
```

Comment:
```suggestion
  constructor({ loginPath, logoutPath, fetchUserl } = { loginPath = '/login', logoutPath = '/logout' }) {
```

Reply:
The former is easier to read than your proposal Tuan, IMHO